ws-nextjs-app/pages/[service]/live/[id]/Stream/index.tsx

```diff
@@ -21,10 +23,18 @@ const Stream = ({
 
   return (
     <div>
-      <Heading css={styles.heading} level={2}>
+      <Heading
+        css={contributors ? styles.heading : styles.headingNoContributors}
+        level={2}
+      >
         Live Reporting
       </Heading>
-      <Paragraph css={styles.subHeading}>By a random dude</Paragraph>
+      {contributors && (
+        <Paragraph role="paragraph" css={styles.subHeading}>
```

Comment:
May be best to use the `data-testid` attribute here, as `role` can have a11y connotations. Something like `data-testid="live-contributors"` would be grand, although with pointing at that in the test instead of the `role` attribute.